doc/coc-toml.txt

```diff
@@ -44,10 +44,10 @@ file name, open `:CocConfig` and write config like this.
 >
   (other configs ...)
   "toml.schema.associations": {
-    ".*/Cargo\\.toml": "taplo://cargo@Cargo.toml",
-    ".*/pyproject\\.toml": "taplo://python@pyproject.toml",
-    ".*/\\.?taplo\\.toml": "taplo://taplo@taplo.toml",
-    ".*/*dein*\\.toml": "file:///home/me/.cache/coc/coc-toml-data/test.json"
+    ".*/Cargo\\.toml": "https://raw.githubusercontent.com/tamasfe/taplo/master/schemas/cargo.toml.json",
```

Comment:
@GopherJ 

> I also noticed that the vim help manual doesn't work for me yet, maybe you can take a look once got time.

I think this is caused by the length of these lines. Each line in doc for vim should be within 78 charactors.

Reply:
We may need to make these lines shortened by omitting url or use dummy one.